Aliases/coffeescript

```diff
@@ -0,0 +1 @@
+../Formula/coffee-script.rb
```

Comment:
I think it should be named `coffeescript` directly since that’s the name upstream uses.